migration/migration_blackbox_test.go

```diff
@@ -546,64 +525,118 @@ func testMigration73(t *testing.T) {
 }
 
 func testMigration74(t *testing.T) {
-	migrateToVersion(sqlDB, migrations[:75], 75)
+	migrateToVersion(t, sqlDB, migrations[:75], 75)
 	assert.True(t, dialect.HasColumn("labels", "border_color"))
 }
 
 func testMigration75(t *testing.T) {
-	migrateToVersion(sqlDB, migrations[:76], 76)
+	migrateToVersion(t, sqlDB, migrations[:76], 76)
 	assert.True(t, dialect.HasIndex("labels", "labels_name_space_id_unique_idx"))
 }
 
 func testMigration76(t *testing.T) {
-	migrateToVersion(sqlDB, migrations[:77], 77)
+	migrateToVersion(t, sqlDB, migrations[:77], 77)
 	assert.False(t, dialect.HasTable("space_resources"))
 	assert.False(t, dialect.HasTable("oauth_state_references"))
 }
 
+func testMigration79(t *testing.T) {
+	migrateToVersion(t, sqlDB, migrations, 79)
+	assert.Nil(t, runSQLscript(sqlDB, "079-old-link-type-relics.sql",
+		space.SystemSpace.String(),
+		link.SystemWorkItemLinkTypeBugBlockerID.String(),
+		link.SystemWorkItemLinkPlannerItemRelatedID.String(),
+		link.SystemWorkItemLinkTypeParentChildID.String(),
+		link.SystemWorkItemLinkCategorySystemID.String(),
+		link.SystemWorkItemLinkCategoryUserID.String(),
+	))
+
+	// When we migrate the DB to version 78 all but the known link types and
+	// categories should be gone, which is what we test below.
+	migrateToVersion(t, sqlDB, migrations, 79)
```

Comment:
I'm aware that this is a bit strange. It is the same line as 544 but I don't know why it is only picked up here.